spec/controllers/admin/members_controller_spec.rb

```diff
@@ -10,7 +10,7 @@
       it "downloads an empty CSV" do
         get :index, format: :csv
 
-        expect(response.body).to eq %[id,full_name,joined_at,email,address,data,created_at,updated_at,token,token_updated_at
+        expect(response.body).to eq %[id,full_name,joined_at,email,address,data,created_at,updated_at
```

Comment:
Is it worth adding a negative assertion, to make sure the response never accidentally leaks the `token`?

Reply:
> Is it worth adding a negative assertion, to make sure the response never accidentally leaks the `token`?

Yes, it so does!

Reply:
Ah, so it does! Nice one 👍